pull_request_template.md

```diff
@@ -25,4 +25,4 @@ e.g.
 * Added topic
 -->
 
-<!-- Please don't forget to request for reviews -->
+<!-- Reviews are automatically requested after the pull request has been created. -->
```

Comment:
```suggestion
<!-- Reviewers from the software team are automatically requested after the pull request has been created. -->
```

Reply:
This is not true, other teams and people can also be requested, check the `CODEOWNERS` files

Reply:
Ah, I see